README.md

```diff
@@ -11,6 +16,47 @@ The software consists of theree parts:
 - [xroad-catalog-persistence](xroad-catalog-persistence/README.md)
   * Library used to persist and read persisted data. Used by both of the above.
   
+![X-Road Catalog overview](xroad-catalog.png)
+
+## Environments
+* Internal development environment in the AWS: [AWS xroad-catalog](http://xroad-catalog.i.palveluvayla.com)
```

Comment:
Would be useful to add the xroad enviroment that is used by the catalog, ie. the UI addresses. ice-cs-i.palveluvayla.com:4000 etc

Reply:
I added XRoad environments aswell.